.gitignore

```diff
@@ -15,3 +15,4 @@ Session.vim
 .netrwhist
 *~
 _build
+rebar.lock
```

Comment:
rebar.lock should probably be added to the git repo, not ignored

Reply:
Oh, I use VScode to format. If no necessary，I close the issue.